Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -1114,4 +1222,143 @@ extension ApolloCodegenConfiguration {
       experimentalFeatures: experimentalFeatures,
       schemaDownloadConfiguration: schemaDownloadConfiguration)
   }
+
+  /// Enum to enable using
+  /// [Automatic Persisted Queries (APQs)](https://www.apollographql.com/docs/apollo-server/performance/apq)
+  /// with your generated operations.
+  ///
+  /// APQs are an Apollo Server feature. When using Apollo iOS to connect to any other GraphQL server,
+  /// `APQConfig` should be set to `.disabled`
+  public enum APQConfig: String, Decodable {
+    /// The default value. Disables APQs.
+    /// The operation document is sent to the server with each operation request.
+    @available(*, deprecated, message: "Use PersistedQueryConfig.disabled instead.")
```

Comment:
I might be missing something but I don't see anywhere in this PR where PersistedQueryConfig is referred to except for these deprecation messages?

Reply:
Yupp. renamed this and missed this message. thanks!